README.md

```diff
@@ -54,46 +54,66 @@ We recommend that you link to `<url for repository's GitHub pages>/workshop/HOME
 
-1. Make sure you have [Bundler](https://bundler.io/) installed. Installation can be accomplished with `gem install bundler`.
- 
-  If you encounter a permissions error, it was solved (using this StackOverflow post as a guide) with the following on Mojave with [Homebrew](https://brew.sh/):
+### Installing GitHub pages dependencies locally
 
-  ```
-  # install rbenv
-  brew update
-  brew install rbenv
+Installing the dependencies for GitHub pages is best done in a separate ruby environment, managed by [rbenv](https://github.com/rbenv/) and [Bundler](https://bundler.io)
 
-  # add to ~/.bash_profile
-  echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
-  ```
+The following instructions are for installation on Mac OS.
+1. [Install rbenv](https://github.com/rbenv/rbenv#installation).
+The easiest, and recommended, installation is through Homebrew:
+```
+# install rbenv
+brew update
+brew install rbenv ruby-build
+```
+Alternatively, you can install with [rbenv-installer](https://github.com/rbenv/rbenv-installer#rbenv-installer) which will work even without Homebrew (though it uses Homebrew if you have it).
 
-  Start a new shell to initialize `rbenv`.
+2. Initialize rbenv and install it for your shell:
+```
+ # run this and follow the printed instructions:
+rbenv init
+```
+Start a new shell to initialize `rbenv`.
 
+3. Install a current stable ruby version
+  ```
+  # install and set up v3.1.3 of Ruby
+  rbenv install 3.1.3
   ```
-  # install and setup v2.7.1 of Ruby
-  rbenv install 2.7.1
-  rbenv shell 2.7.1
-  rbenv rehash
-  ``` 
 
-  Now you're ready to run `gem install bundler`. 
+4. Activate your ruby version. Here you have a few options:
 
-2. Install Jekyll (`gem install jekyll`).
-3. Run `bundle install` from the root of this repository, where the `Gemfile` is located.
-4. Run `bundle exec jekyll serve` and navigate to the server address.
+   a. If you want to set the global ruby version that will be used wherever you use `ruby`, you can use
+   ```
+   rbenv global 3.1.3
+   ```
 
-If you've already followed the steps above and are in a new shell, run the following steps to build locally:
+   b. To only use this version of ruby within the repository only, navigate to the root of this repository and use:
+   ```
+   rbenv local 3.1.3
+   ```
+   This will create a `.ruby-version` file that will automatically activate this version of ruby whenever you are in this directory.
 
+5. Now you should be ready to install Bundler:
 ```
-rbenv shell 2.7.1
-rbenv rehash
-bundle exec jekyll serve  
+gem install bundler
 ```
+
+6. Finally, run `bundle install` from the root of this repository, where the `Gemfile` is located.
+This will install all additional dependencies.
+
+### Running a local jekyll server
+
+Once all dependencies are installed, you should be able to start a local jekyll server with:
+
+```
+bundle exec jekyll serve
```

Comment:
At this point my `which jekyll` is still heading to an older `/usr/local/bin/jekyll`. Before we get here, I think it's worth adding a step `rbenv rehash` which will clean up all the executables to default to the `rbenv` ones. ("fixed" my path, `which jekyll` --> `/Users/sjspielman/.rbenv/shims/jekyll`).

But either way I'm getting a different build error: 
```
/Users/sjspielman/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/jekyll-3.9.2/lib/jekyll/commands/serve/servlet.rb:3:in `require': cannot load such file -- webrick (LoadError)
```

I looked into it and it seems that `webrick` no longer comes with ruby >=3 but has to be installed as a gem. I ran `bundle add webrick` to update the gemfile, and then it served without error!

In conclusion, **add `webrick` to the Gemfile**. Not sure of version though tbh...

Reply:
I did add webrick!

The version of jekyll you get with `which jekyll` does not matter if you use `bundle exec jekyll serve`. The point of that command is to use the bundled jekyll regardless of what else might be installed.

Reply:
Ah womp, sorry, wasn't in the right branch. All set :)

Reply:
I also received the same original error noted above in https://github.com/AlexsLemonade/training-specific-template/pull/101/files#r1101582245 and was able to get things working by running `bundle add webrick` as Stephanie mentioned then re-running this step!

Reply:
Just to confirm, you hadn't pulled this branch for testing, right? Because I did add webrick to the gemfile as part of this PR: https://github.com/AlexsLemonade/training-specific-template/blob/b3ca78851811ab3f08df78d7d41234066b7019d0/Gemfile#L4